apps/examples/testcase/le_tc/kernel/tc_libc_sched.c

```diff
@@ -90,6 +123,8 @@ int libc_sched_main(void)
 {
 	tc_libc_sched_sched_get_priority_max();
 	tc_libc_sched_sched_get_priority_min();
-
+#ifndef CONFIG_CANCELLATION_POINTS
```

Comment:
task_setcanceltype() API is compiled only when 
ifneq ($(CONFIG_CANCELLATION_POINTS),y)
Hence, it should be ifndef